README.md

```diff
@@ -1 +1,57 @@
-# ci-script
\ No newline at end of file
+# ci-script
+
+Script to help setup a repo with continuous integration support.
+
+- Uses GitHub Actions.
+- Actions configs for different setups.
+
+## Usage
+
+From your source dir, run:
+
+    ../ci-script/setup-ci MODE
```

Comment:
I'll just say that debian based distros make it easy to setup a ~/bin folder that will automatically be put on the path and it's easy to put a symlink in there for scripts like this.  Same applies to `pubnpm` script.

Reply:
Sure, I was trying to be minimal.  I'll add a short note.  Safe to assume some OS knowledge here.